public/main.js

```diff
@@ -0,0 +1,57 @@
+
+let mymap = L.map('mapid').setView([44.476264, -73.212883], 16)
+
+
+L.tileLayer('https://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/{z}/{y}/{x}', {
+    minZoom: 15,
+    maxZoom: 17,
+    attribution: 'Tiles &copy; Esri &mdash; Source: Esri, DeLorme, NAVTEQ, USGS, Intermap, iPC, NRCAN, Esri Japan, METI, Esri China (Hong Kong), Esri (Thailand), TomTom, 2012',
+    id: 'mapbox.streets'
+}).addTo(mymap);
+
+window.fetch('https://json-server.burlingtoncodeacademy.now.sh/restaurants')
+    .then(response => response.json())
+    .then((json) => {
+        const target = document.getElementById("restaurantList");
+
+        const postList = document.createElement("ul")
+
+        const posts = json.map((postData) => {
+            const listItem = document.createElement("li")
+            let link = listItem.appendChild(document.createElement("a"))
+            link.textContent = `${postData.name}`; //populates list
+            link.href = `${postData.website}` //links to restaurant's website
+            
+            placeMarker(postData)
+            return listItem;
+
+        });
+
+        posts.forEach((post) => {
+            postList.appendChild(post);
+        })
+
+        target.append(postList);
+
+        return undefined;
```

Comment:
Since you are just using this information to populate your restaurant list, you don't need to return undefined here. Unless you are getting information from a fetch that you will need to use later in your code, then it is fine to not return a value out of your fetch. That being said, it may be a good idea to return some restaurant data from here that you get from your fetch.